Utilities/CodeGeneration/EdFi.Ods.CodeGen/Options.cs

```diff
@@ -14,16 +14,30 @@ public class Options
         [Option('r', "repositoryPath", Required = false, HelpText = "Path to the code repository where the ODS is located.")]
         public string CodeRepositoryPath { get; set; }
 
-        [Option('e', "engine", Required = false, HelpText = "Database engine type: SQLServer or PostgreSQL.", Default = EngineType.SQLServer)]
+        [Option(
+            'e', "engine", Required = false, HelpText = "Database engine type: SQLServer or PostgreSQL.",
+            Default = EngineType.SQLServer)]
         public EngineType Engine { get; set; }
 
-        [Option('v', "viewsFromDatabase", Required = false, HelpText = "Request view metadata from an existing empty database.", Default = false)]
+        [Option(
+            'v', "viewsFromDatabase", Required = false, HelpText = "Request view metadata from an existing empty database.",
+            Default = false)]
         public bool ViewsFromDatabase { get; set; }
 
-        [Option('i', "includePlugins", Required = false, HelpText = "Flag to determine if plugin assemblies are included", Default = false)]
+        [Option(
+            'i', "includePlugins", Required = false, HelpText = "Flag to determine if plugin assemblies are included",
+            Default = false)]
         public bool IncludePlugins { get; set; }
 
-        [Option("extensionPaths", Required = false, HelpText = "Array of paths for the Extension location to determine if plugin assemblies are included",Separator =',' )]
+        [Option(
+            "extensionPaths", Required = false,
+            HelpText = "Array of paths for the Extension location to determine if plugin assemblies are included",
+            Separator = ',')]
         public IEnumerable<string> ExtensionPaths { get; set; }
+
+        [Option(
+            't', "tupletableFromDatabase", Required = false, HelpText = "Request tuple table metadata from an existing empty database.",
```

Comment:
I think overall this has ended up a lot more complicated than we want. Our goal is to reduce dependencies on the database where we can, particularly in code gen.  You've had to make major changes to code gen and introduced a lot of dependencies, all to generate a mapping with just two columns.  I'm fine with the approach you're taking of a separate entity and nHibernate mapping file to maintain access for this tuple table, but my recommendation would for them to just be hard coded, and not try to generate them.  This table is fairly stable, so I'm not sure we need all this complexity to generate it.  On top of that even with generation you're still requiring a very manual step to generate it, so I don't think its paying off.  I'd rip all this code gen for those artifacts and just have them checked in and we keep them in sync if we have to change the tuple table down the road.  CC: @vimayya